services/movie_session.py

```diff
@@ -0,0 +1,73 @@
+from datetime import datetime
+from typing import Tuple
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+
+) -> MovieSession:
+    new_movie_session = MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+
+    return new_movie_session
+
+
+def get_movies_sessions(
+        session_date: str = None
+) -> QuerySet:
```

Comment:
ou could indicate that you return a queryset of MovieSession instances in the type annotation.